field_audit/models.py

```diff
@@ -216,10 +216,11 @@ class Meta:
         constraints = [
             models.CheckConstraint(
                 name="field_audit_auditevent_chk_create_or_delete_or_bootstrap",
-                check=~(
+                check=models.Q(
```

Comment:
I ran `sqlmigrate` on the existing migration and the autogenerated migration by Django 4.2 to compare.

Old
```
CHECK (NOT ((("is_create" AND "is_delete") OR ("is_bootstrap" AND "is_create") OR ("is_bootstrap" AND "is_delete"))))
```

New
```
CHECK (NOT (("is_create" AND "is_delete") OR ("is_bootstrap" AND "is_create") OR ("is_bootstrap" AND "is_delete")))
```

So the difference is just an extra parenthesis around the inner condition of the `NOT`, but is that enough of a difference to warrant not changing the existing migration file itself?

Reply:
Semantically that is no different. I think it would be fine to change the migration file if it does not cause Django 3.2 to want to create a new migration.

Reply:
Cool. See 0bb2fc9b1a859bfad63b368db5a59e0fae51357d. I tested that `makemigrations` on both 3.2 and 4.2 does not result in a new migration being created.